subprojects/docs/src/samples/java/jvm-multi-project-with-code-coverage/groovy/code-coverage-report/build.gradle

```diff
@@ -3,5 +3,5 @@ plugins {
 }
 
 dependencies {
-    implementation project(':application')
+    jacocoAggregation project(':application')
```

Comment:
Should we take this path further, we should also change this sample to not need the `code-coverage-report` subproject at all and use the existing `application` project for aggregation.